test/components/structures/__snapshots__/ThreadPanel-test.tsx.snap

```diff
@@ -45,29 +45,55 @@ exports[`ThreadPanel Header expect that My filter for ThreadPanelHeader properly
 `;
 
 exports[`ThreadPanel Header expect that ThreadPanelHeader has the correct option selected in the context menu 1`] = `
-<AccessibleButton
+<RovingAccessibleButton
   aria-checked={true}
   className="mx_ThreadPanel_Header_FilterOptionItem"
-  element="div"
   onClick={[Function]}
   role="menuitemradio"
-  tabIndex={-1}
 >
-  <div
+  <AccessibleButton
```

Comment:
This looks wrong, button inside a button. Both having aria-checked and the same className?

Reply:
The only actual DOM is the `<div>`. The `props` are just getting spread over everything as we go down the chain, `{...props}` 

```
<div id="mx_ContextualMenu_Container">
  <div class="mx_ContextualMenu_wrapper mx_ThreadPanel__header" style="top: 100px; right: 33px;">
    <div class="mx_ContextualMenu_background"></div>
    <div class="mx_ContextualMenu mx_ContextualMenu_withChevron_top" role="menu">
      <div class="mx_ContextualMenu_chevron_top"></div>
      <div role="menuitemradio" aria-checked="false" tabindex="0" class="mx_AccessibleButton mx_ThreadPanel_Header_FilterOptionItem">
        <span>My threads</span>
        <span>Shows all threads you've participated in</span>
      </div>
      <div role="menuitemradio" aria-checked="true" tabindex="-1" class="mx_AccessibleButton mx_ThreadPanel_Header_FilterOptionItem">
        <span>All threads</span>
        <span>Shows all threads from current room</span>
      </div>
    </div>
  </div>
</div>
```

Reply:
Oh I see, 🤦